test/testdrive/avro-ocf.td

```diff
@@ -36,6 +36,25 @@ a          NO        int8
 b          NO        int4
 mz_obj_no  NO        int8
 
+$ avro-ocf-write path=data-no-codec.ocf schema=${writer-schema} disable-codec=true
```

Comment:
If you take my suggestion above, then `disable-codec` could become `codec`, and take one of `<omit, null, deflate>`, and we could even test deflated Avro OCF files. I'm somewhat leery of introducing these one-off options like `disable-codec` to testdrive commands, since it indicates we've probably got the abstraction boundary a bit wrong—but `codec` is general-purpose enough to assuage my concerns there.

Reply:
Why not just `codec=<null, deflate>`, and if you don't pass a `codec` argument at all, it will be omitted?

Reply:
For the record, I'm perfectly comfortable if we only test one or the other (we're literally testing the presence of one `unwrap_or`). But the whole point of this hunk, and all the changes to `Writer`, is testing `null` vs `<omitted>`, no?

Reply:
Well yeah, I'm just saying to test `null` you would pass `codec=null`, and to test omitted, you would not pass any `codec` argument.

Reply:
Oh lol, I totally misunderstood. That makes sense.